docker/ubuntu16.04/Dockerfile

```
@@ -0,0 +1,20 @@
+FROM ubuntu:16.04
+
+ARG CNTOOLKIT_VERSION="2.5.2-1"
+ENV maintainer="cnnl@cambricon.com"
+ENV NEUWARE_HOME="/usr/local/neuware/"
+
+WORKDIR /tmp
+
+RUN apt update && apt install --no-install-recommends -y git apt-transport-https wget build-essential cmake python3 python pkg-config libprotobuf-dev libxml2-dev protobuf-compiler && apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# NOTE: the url below is only for internal usage
+RUN wget http://daily.software.cambricon.com/release/cntoolkit/Linux/x86_64/Ubuntu/16.04/${CNTOOLKIT_VERSION}/cntoolkit_${CNTOOLKIT_VERSION}.ubuntu16.04_amd64.deb
+# You could use COPY with your own cntoolkit package
+
+RUN dpkg -i cntoolkit_${CNTOOLKIT_VERSION}.ubuntu16.04_amd64.deb && \
```

```suggestion
RUN apt-get install -y /tmp/cntoolkit_${CNTOOLKIT_VERSION}.ubuntu16.04_amd64.deb && \
```

一般更偏好 apt-get，dpkg是底层命令。

---

done